res/css/structures/_RoomStatusBar.scss

```diff
@@ -76,6 +76,8 @@ limitations under the License.
         min-height: 70px;
         margin: 12px;
         padding-left: 16px;
+        padding-top: 4px;
+        padding-bottom: 4px;
```

Comment:
Added so the status bar has a little bit more consistent padding when it's squished at smaller widths. It was kinda relying only on the `min-height` before.

The styles here could use some love in general but we can iterate that in a separate PR.

Before | After
--- | ---
<img width="483" alt="" src="https://user-images.githubusercontent.com/558581/170393169-892f5064-ec67-4377-b333-0535ecd06505.png"> | <img width="481" alt="" src="https://user-images.githubusercontent.com/558581/170393171-4a3d87af-1060-417c-a877-1eb16dc5e13e.png">